website/project/model.py

```diff
@@ -3335,8 +3335,8 @@ def add_unregistered_contributor(self, fullname, email, auth, send_email='defaul
         :returns: The added contributor
```

Comment:
Update the parameters to have the full set for the current function. It's your new parameter plus probably some of the others.